### PR TITLE
[TF-666] Web Settings UI Scaling

### PR DESCRIPTION
**Pull request by cartmouse**
## Summary

Update the Web Settings to be reactive to various resolutions.

### Tests Added

No tests added as we are not currently doing unit testing for UI.

## Contributor Tasks

_These tasks are for the pull request creator to tick off._

- [ ] Developer testing
- [ ] PO review (optional depending on work type)
- [ ] XDR review (optional depending on work type)
- [ ] QA review (or another developer if no QA is available)
- [ ] Ensure documentation requirements are met e.g., public API is commented
- [ ] Consider any licensing/other legal implications e.g., notices required

If there is an associated JIRA issue:
- [ ] Include a link to the JIRA issue in the summary above
- [ ] Update the release notes on the issue
- [ ] Make sure the fix version on the issue is set correctly

## Reviewer Tasks

_Add any instructions or tasks for the reviewer such as specific test considerations before this can be merged._

- [ ] Code reviewed
- [ ] Non-code assets reviewed
- [ ] Documentation reviewed - includes checking documentation requirements are met and not missing e.g., public API is commented

### CODE REVIEW

File: TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
 import { InteractionType } from 'TouchFree/src/TouchFreeToolingTypes';
 
 import { InteractionExplainer } from '@/Videos';
 
 import { DocsLink, LabelledToggleSwitch, RadioGroup, RadioLine, Slider, TextSlider } from '@/Components';
+import { MiscTextButton } from '@/Components/TFButton/TFButtons';
 
 import { DefaultInteractionConfig } from './SettingsTypes';
 
+const classes = classnames.bind(styles);
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx:32]: 💬 Why did you bind classnames here but not in other files?

File: TF_Settings_Web/src/Components/TFButton/TFButtons.tsx [hunk 2]
```diff
@@ -83,35 +86,58 @@ export const HorizontalIconTextButton: React.FC<IconTextButtonProps> = ({
...
+        <img style={iconStyle} src={icon} alt={alt} />
+    </BaseTFButton>
+);
+
+interface MiscTextButtonProps extends BaseButtonProps {
+    title: string;
+}
 
-    return BaseTFButton(`${className} horizontal-tf-button`, canHover, onClick, content, buttonStyle);
+export const MiscTextButton: React.FC<MiscTextButtonProps> = ({ buttonStyle, title, titleStyle, onClick }) => {
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Components/TFButton/TFButtons.tsx]: :eyes: Should this be exported via index.ts

File: TF_Settings_Web/src/Components/DocsLink/DocsLink.module.scss [hunk 3]
```diff
@@ -12,13 +13,18 @@
 }
 
 .docs-modal {
-    $height: 25%;
+    --height: 25vh;
+    --width: 80vw;
+    @media (orientation: landscape) {
+        --height: 50vh;
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Components/DocsLink/DocsLink.module.scss:19]: 💬 Out of interest does this not work with $ scss variables (to avoid using var()). Remember running into something with that and the order the variable gets updated

File: TF_Settings_Web/src/Components/TFButton/TFButtons.tsx (hunk 2 above)
Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Components/TFButton/TFButtons.tsx]: :shrug: Can also get rid of the return statemennt

Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Components/TFButton/TFButtons.tsx]: :shrug: Is there a better name to describe this than misc button? OutlinedTextButton? :shrug:

File: TF_Settings_Web/src/Components/Radio/RadioGroup.tsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import './RadioButtons.scss';
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Components/Radio/RadioGroup.tsx:1]: 💬 This class still relies on input[type='radio'] selector from RadioButtons.module.scss but can't tell that from the imports

I think a better improvement would be to make those input selectors into classes so they apply more specifically but might be out of scope

File: TF_Settings_Web/src/Components/Radio/RadioLine.tsx [hunk 5]
```diff
@@ -35,7 +41,7 @@ export class RadioLine extends React.Component<RadioProps, {}> {
                             checked={this.props.selected === index}
                         />
                         <span className="checkmark" />
-                        <p className="sliderLabel"> {option} </p>
+                        <p className={sliderClasses['sliderLabel']}> {option} </p>
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Components/Radio/RadioLine.tsx]: :shrug: Should sliderLabel be moved to a more general location? As isn't specific to sliders (maybe to interactionsStyles)

File: TF_Settings_Web/src/Components/TextEntry/TextEntry.module.scss [hunk 6]
```diff
@@ -35,28 +33,26 @@
...
     font-family: v.$base-font;
     color: white;
 
     display: flex;
-    flex-direction: row;
     align-items: center;
 
-    margin: 0.75rem;
+    margin: 0.75rem 0;
     background-image: linear-gradient(180deg, #353535, #2d2d2d), linear-gradient(180deg, #373737 0%, #2c2c2c 100%);
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Components/TextEntry/TextEntry.module.scss]: :shrug: This looks similar to some of the gradiants we have in variables. Could it be converted?

File: TF_Settings_Web/src/Components/TextEntry/TextEntry.module.scss [hunk 7]
```diff
@@ -35,28 +33,26 @@
...
 
-    min-height: 5rem;
-    max-height: 5rem;
-}
+    height: 5rem;
 
-.textEntryBackgroundLabel.textEntryBackgroundLabelSelected {
-    background-image: linear-gradient(180deg, #353535, #2d2d2d), linear-gradient(180deg, #00eb86, #00cdcf);
+    &--selected {
+        background-image: linear-gradient(180deg, #353535, #2d2d2d), linear-gradient(180deg, #00eb86, #00cdcf);
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Components/TextEntry/TextEntry.module.scss]: :shrug: Looks the same linear-grad as the support button - could this style be added to variables and then imported in both places?

File: TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx (hunk 1 above)
Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx:32]: Can't see a pattern to binding / not binding throughout the different files

File: TF_Settings_Web/src/App.module.scss [hunk 8]
```diff
@@ -3,29 +3,19 @@
...
 }
 
-.title-line {
+@mixin title-line {
     color: white;
     text-align: left;
-    padding-left: 3rem;
     display: flex;
     flex-direction: row;
     align-items: left;
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/App.module.scss]: :shrug: From dev-tools this is an invalid property and doesn't seem to do anything (know it was already here but can remove as part of this PR?)

File: TF_Settings_Web/src/App.tsx [hunk 9]
```diff
@@ -39,9 +39,9 @@ const App: React.FC = () => {
     }, []);
 
     return (
-        <div className="app">
+        <div className={classes['app']}>
             <ControlBar tfStatus={tfStatus} touchFreeVersion={touchFreeVersion} />
-            <div className="page-content">
+            <div className={classes['page-content']}>
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/App.tsx]: 💬 I don't think this div is needed anymore? All it does is set the background color which is the same as the app class above. I think before it had a height etc so child could use % but from testing removing this doesn't seem to make a difference

File: TF_Settings_Web/src/Pages/Camera/QuickSetup/QuickSetup.module.scss [hunk 10]
```diff
@@ -0,0 +1,19 @@
+@use '@/App.module.scss';
+
+.container {
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Pages/Camera/QuickSetup/QuickSetup.module.scss]: I couldn't see  where this is used. Also this file is only used for PositionSelectionScreen so would that be a better name?

File: TF_Settings_Web/src/Pages/Camera/QuickSetup/Calibration/CalibrationManager.tsx [hunk 11]
```diff
@@ -1,4 +1,4 @@
-import './Calibration.scss';
+import './Calibration.module.scss';
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Pages/Camera/QuickSetup/Calibration/CalibrationManager.tsx]: 💬 Is this import needed?

File: TF_Settings_Web/src/Pages/Camera/QuickSetup/Calibration/CalibrationCompleteScreen.tsx [hunk 12]
```diff
@@ -1,4 +1,4 @@
-import './Calibration.scss';
+import classes from './Calibration.module.scss';
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Pages/Camera/QuickSetup/Calibration/CalibrationCompleteScreen.tsx]: The setup screen feels very low down with the practice button being pretty much at the bottom. There's a lot of wasted space at the top and between the elements

File: TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+        p {
+            padding: 0;
+            text-align: center;
+            line-height: 40px;
+        }
+    }
+
+    &--misc {
+        width: 25vw;
+        height: 4rem;
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss:59]: 💬 Is rem for height intended?

File: TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+        }
+
+        padding: 0.3rem;
+
+        background-image: linear-gradient(180deg, #353535, #2d2d2d), linear-gradient(180deg, #00eb86, #00cdcf);
+        background-clip: content-box, border-box;
+        border-radius: 50px;
+
+        > h1 {
+            font-size: 1.5rem !important;
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss]: 💬 Is there anyway to avoid !important? Moiving the default > h1 {} above this might solve? Or can use a more specific class setup

File: TF_Settings_Web/src/Components/TFButton/TFButtons.tsx [hunk 15]
```diff
@@ -83,35 +86,58 @@ export const HorizontalIconTextButton: React.FC<IconTextButtonProps> = ({
...
+    buttonStyle,
+    children,
+}) => {
     const [hovered, setHovered] = React.useState<boolean>(false);
     const [pressed, setPressed] = React.useState<boolean>(false);
 
     return (
         <button
-            className={`${buttonClass} tf-button ${
-                canHover && hovered ? `tf-button--hovered ${buttonClass}--hovered` : ''
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Components/TFButton/TFButtons.tsx:113]: 💬 We've lost the ability to add --hovered and --pressed styles to the buttonClass we pass in

File: TF_Settings_Web/src/Components/TFButton/TFButtons.tsx [hunk 16]
```diff
@@ -83,35 +86,58 @@ export const HorizontalIconTextButton: React.FC<IconTextButtonProps> = ({
...
-            } ${pressed ? `tf-button--pressed ${buttonClass}--pressed` : ''}`}
+            className={classes(buttonClass, 'tf-button', `tf-button--${type}`, {
+                'tf-button--hovered': canHover && hovered,
+                [`tf-button--${type}--hovered`]: canHover && hovered,
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Components/TFButton/TFButtons.tsx:157]: 🙈 This has the same logic as the one above - can these be combined

File: TF_Settings_Web/src/Components/TFButton/TFButtons.tsx (hunk 2 above)
Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Components/TFButton/TFButtons.tsx]: :eyes: Should also set className on the misc button (it's already included in it's props)

File: TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx [hunk 17]
```diff
@@ -1,18 +1,26 @@
...
+    InteractionConfig,
+    InteractionConfigFull,
+    TrackedPosition,
+} from 'TouchFree/src/Configuration/ConfigurationTypes';
 import { InteractionType } from 'TouchFree/src/TouchFreeToolingTypes';
 
 import { InteractionExplainer } from '@/Videos';
 
 import { DocsLink, LabelledToggleSwitch, RadioGroup, RadioLine, Slider, TextSlider } from '@/Components';
+import { MiscTextButton } from '@/Components/TFButton/TFButtons';
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx]: 🙈 Import from @/Components

File: TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx [hunk 18]
```diff
@@ -24,351 +32,252 @@ const TouchPlaneTrackingOptions: Record<string, TrackedPosition> = {
...
+        );
 
-    onInteractionMaxDistChange(newValue: number): void {
-        this.setState((state) => {
-            const newState = this.getStateCopy(state);
-            newState.interactionConfig.InteractionMaxDistanceCm = newValue;
-            return newState;
-        });
-    }
+        const onResize = () => {
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx]: :shrug: Could be one line

File: TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx [hunk 19]
```diff
@@ -1,18 +1,26 @@
-import './Interactions.scss';
+import classnames from 'classnames/bind';
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx:1]: 👍 This is a really nice refactor. What a joy to review. Down with classes!

Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx:1]: and nice use of a reducer

File: TF_Settings_Web/src/Pages/Camera/Masking/sceneRendering.ts [hunk 20]
```diff
@@ -72,6 +72,11 @@ const dummy = new Object3D();
 const BASE_LINE_THICKNESS = 0.005;
 
 export const setupRenderScene = (div: HTMLDivElement) => {
+    if (div.children.length > 0) {
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Pages/Camera/Masking/sceneRendering.ts]: 💬 Why is this needed?

File: TF_Settings_Web/src/Pages/Camera/Masking/MaskingScreen.tsx [hunk 21]
```diff
@@ -170,13 +182,13 @@ const MaskingScreen: React.FC = () => {
...
                     maskingValue={sliderInfo[1]}
-                    canvasInfo={{ size: 800, topOffset: 50, leftOffset: 100 }}
+                    canvasSize={innerHeight < innerWidth ? innerHeight * 0.45 : innerWidth * 0.96 - innerHeight * 0.1}
                     clearMasking={clearMasking}
                     onDrag={setMasking}
                     onDragEnd={sendMaskingRequest}
                 />
             )),
-        [masking.current]
+        [masking.current, innerHeight]
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Pages/Camera/Masking/MaskingScreen.tsx]: 💬 Do you need to add innerWidth to dependencies as well?

File: TF_Settings_Web/src/Pages/Camera/Masking/MaskingSlider.tsx [hunk 22]
```diff
@@ -36,31 +35,22 @@ export const MaskingSlider: React.FC<MaskingSliderProps> = ({ direction, masking
     // ===== UseEffects =====
     useEffect(() => {
         if (!sliderRef.current) return;
-        const top = Number.parseFloat(getComputedStyle(sliderRef.current).top);
-        const left = Number.parseFloat(getComputedStyle(sliderRef.current).left);
+        const size = innerHeight < innerWidth ? innerHeight * 0.45 : innerWidth * 0.96 - innerHeight * 0.1;
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Pages/Camera/Masking/MaskingSlider.tsx]: 💬 This is the same 'magic value' as in MaskingScreen. Should this be stored somewhere

File: TF_Settings_Web/src/Pages/Camera/Masking/MaskingSlider.tsx [hunk 23]
```diff
@@ -36,31 +35,22 @@ export const MaskingSlider: React.FC<MaskingSliderProps> = ({ direction, masking
...
+        const top = direction === 'lower' ? size - 5 : 0;
+        const left = direction === 'right' ? size - 5 : 0;
 
-        setPosition(direction, maskingValue, canvasInfo.size, { top, left }, sliderRef.current);
+        setPosition(direction, maskingValue, canvasSize, { top, left }, sliderRef.current);
         setInitialSliderPos({ top, left });
-    }, []);
+    }, [innerHeight]);
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Pages/Camera/Masking/MaskingSlider.tsx]: 💬 innerWidth here as well?

File: TF_Settings_Web/src/Pages/Camera/Masking/MaskingSlider.tsx (hunk 22 above)
Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Pages/Camera/Masking/MaskingSlider.tsx]: Is this not the same as canvasSize?

File: TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx (hunk 1 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx:32]: I've only been using it where I need the "conditional classes" feature. Could put it in everywhere if you'd prefer that?

Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx:32]: My preference would be to bind everywhere for consistency and incase you need add conditional classes later. This would add a few lines of bloat to every file though. Could we make a nicer hook of some sort....?

File: TF_Settings_Web/src/Components/DocsLink/DocsLink.module.scss (hunk 3 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Components/DocsLink/DocsLink.module.scss:19]: It doesn't 😞 Had to use css variables for it

File: TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx (hunk 19 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx:1]: Thank you 😁 Goodbye classes!

File: TF_Settings_Web/src/Pages/Camera/Masking/sceneRendering.ts (hunk 20 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Pages/Camera/Masking/sceneRendering.ts]: It's not! It'll be removed

File: TF_Settings_Web/src/Pages/Camera/Masking/MaskingSlider.tsx (hunk 23 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Pages/Camera/Masking/MaskingSlider.tsx]: Updated to use the `isLandscape` hook instead

File: TF_Settings_Web/src/Pages/Camera/Masking/MaskingSlider.tsx (hunk 22 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Pages/Camera/Masking/MaskingSlider.tsx]: Yep! Updated to use that instead

File: TF_Settings_Web/src/Pages/Camera/Masking/MaskingScreen.tsx (hunk 21 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Pages/Camera/Masking/MaskingScreen.tsx]: Now uses `isLandscape` hook

File: TF_Settings_Web/src/Components/Radio/RadioLine.tsx (hunk 5 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Components/Radio/RadioLine.tsx]: Moved!

File: TF_Settings_Web/src/Components/Radio/RadioGroup.tsx (hunk 4 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Components/Radio/RadioGroup.tsx:1]: I've done a bit of an update to the whole RadioButtons section

File: TF_Settings_Web/src/Components/TFButton/TFButtons.tsx (hunk 2 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Components/TFButton/TFButtons.tsx]: I think I've address all these!

File: TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx (hunk 1 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx:32]: I'll go through and do this now

File: TF_Settings_Web/src/Pages/Camera/QuickSetup/QuickSetup.module.scss (hunk 10 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Pages/Camera/QuickSetup/QuickSetup.module.scss]: Will change the name! `container` is the top level of the position selection screen

File: TF_Settings_Web/src/Pages/Camera/QuickSetup/Calibration/CalibrationManager.tsx (hunk 11 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Pages/Camera/QuickSetup/Calibration/CalibrationManager.tsx]: Nope, will remove

File: TF_Settings_Web/src/Pages/Camera/QuickSetup/Calibration/CalibrationCompleteScreen.tsx (hunk 12 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Pages/Camera/QuickSetup/Calibration/CalibrationCompleteScreen.tsx]: This should be addressed, let me know what you think now it's been changed!

File: TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss (hunk 13 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss:59]: It gives a nice, consistent size so I'd be tempted to leave it

File: TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss (hunk 14 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss]: `!important` is required here because it needs to overwrite the base class' `h1` styling. Could have a look at more specific classes, but would take a bit more time

File: TF_Settings_Web/src/Components/TFButton/TFButtons.tsx (hunk 15 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Components/TFButton/TFButtons.tsx:113]: Yes, I tried to keep it but it's just so complicated with modules. I think it would be possible with modules, but it'd be really messy

File: TF_Settings_Web/src/Components/TFButton/TFButtons.tsx (hunk 16 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Components/TFButton/TFButtons.tsx:157]: I believe to get the right module code appended, they have to be different

File: TF_Settings_Web/src/Components/TFButton/TFButtons.tsx [hunk 24]
```diff
@@ -83,35 +89,74 @@ export const HorizontalIconTextButton: React.FC<IconTextButtonProps> = ({
...
 };
 
-export const BaseTFButton = (
-    buttonClass: string,
-    canHover: boolean,
-    onClick: (event: TFClickEvent) => void,
-    content: JSX.Element,
-    buttonStyle?: CSSProperties
-) => {
+interface MiscTextButtonProps extends BaseButtonProps {
```
Comment (JordanBerni-UL) [inline TF_Settings_Web/src/Components/TFButton/TFButtons.tsx]: 🙈 Update props name to match new button name

File: TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss (hunk 14 above)
Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss]: Moving the h1{} block in .tf-button above the override h1 stlying meaning you don't need important as styles lower down in the css file overwrite those above (at the moment the h1{} block is below the --misc { >h1} so the default styling gets priority)

File: TF_Settings_Web/src/Components/TFButton/TFButtons.tsx (hunk 15 above)
Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Components/TFButton/TFButtons.tsx:113]: Ah true - forgot about modules. Can tackle this when we need it

File: TF_Settings_Web/src/Components/TFButton/TFButtons.tsx (hunk 16 above)
Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Components/TFButton/TFButtons.tsx:157]: Tried playing around with this but couldn't get it working due to the modules (like you said)

File: TF_Settings_Web/src/Pages/Camera/Masking/sceneRendering.ts (hunk 20 above)
Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Pages/Camera/Masking/sceneRendering.ts]: Could a comment be added as for why that block of code is needed

File: TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss (hunk 14 above)
Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss]: Think there are some other cases of !important in that file which can be solved in the same way (keen on avoiding !important unless it's overwriting a third party stylesheet and is unavoidable)

File: TF_Settings_Web/src/Pages/Camera/Masking/sceneRendering.ts (hunk 20 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Pages/Camera/Masking/sceneRendering.ts]: Added!

File: TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss (hunk 14 above)
Comment (cartmouse) [reply TF_Settings_Web/src/Components/TFButton/TFButtons.module.scss]: All removed!

**Review by JordanBerni-UL — APPROVED** [review]
